### PR TITLE
fix(CI): Tests Android description

### PR DESCRIPTION
**Pull request by workgroupengineering**
GitHub Issue (If applicable): closes #

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: build/ci/.azure-devops-android-tests.yml [hunk 1]
```diff
@@ -142,6 +142,8 @@ jobs:
           FAILBUILD_ON_FAILURE: true
           ALLOW_RERUN: true
           UITEST_TEST_TIMEOUT: '270000'
+          ANDROID_VERSION: '9.0'
+          JobDiplayName: 'Android $(ANDROID_VERSION) api $(ANDROID_SIMULATOR_APILEVEL) $(TARGETPLATFORM_NAME) UI Tests Group $(UNO_UITEST_BUCKET_ID)'
```
Comment (Youssef1313) [inline build/ci/.azure-devops-android-tests.yml:146]: I wouldn't introduce a new variable here just to use in display name

Why not something like this?

```suggestion
          JobDiplayName: 'Android 9.0 api $(ANDROID_SIMULATOR_APILEVEL) $(TARGETPLATFORM_NAME) UI Tests Group $(UNO_UITEST_BUCKET_ID)'
```

**Comment by Youssef1313** [comment]
This is currently broken:

image

---

This is how it used to look:

image

**Review by Youssef1313 — COMMENTED** [review]
It's quite unclear for me what the intent here is.

Is your intent to "refactor" the YML by introducing the new `JobDisplayName`, or is it to update some descriptions?

**Comment by workgroupengineering** [comment]
> It's quite unclear for me what the intent here is.
> 
> Is your intent to "refactor" the YML by introducing the new `JobDisplayName`, or is it to update some descriptions?

The main reason is this

immagine

Second fix job description

File: build/ci/.azure-devops-android-tests.yml [hunk 2]
```diff
@@ -110,7 +110,6 @@ jobs:
       ArtifactType: Container
 
 - job: Android_Tests
-  displayName: 'Run Android UI Tests'
```
Comment (Youssef1313) [inline build/ci/.azure-devops-android-tests.yml:113]: Why is this removed?

Comment (workgroupengineering) [reply build/ci/.azure-devops-android-tests.yml:113]: it is a test of what is displayed without.

Comment (Youssef1313) [reply build/ci/.azure-devops-android-tests.yml:113]: I think it will display as "Android_Tests" (the job name)

**Comment by jeromelaban** [comment]
Thanks for your contribution. Recent changes in CI have made this changes deprecated. Closing.
